### ISSUE TITLE
Question about matched state?

### ISSUE THREAD

**Issue by vfcerexwn** [StanfordMIMI/DDM2#27]
Can I ask what is the correct shape for matched state? I Originally thought it should be an integer referred to t, but it's not correct.

**Comment by tiangexiang** [comment]
Yes, the matched state should be an integer for one single slice. You can refer to the parsing codes [here](https://github.com/StanfordMIMI/DDM2/blob/main/data/mri_dataset.py#L40), where a dictionary is constructed to store integer matched states for all volumes and all slices
